app/cafe.py

```diff
@@ -0,0 +1,25 @@
+from app.errors import NotVaccinatedError
+from app.errors import OutdatedVaccineError
+from app.errors import NotWearingMaskError
+
```

Comment:
There is no need to add two blank lines between different import groups, one is enough